recipes/qtlseq/meta.yaml

```diff
@@ -24,7 +24,9 @@ requirements:
     - python >=3.5
     - samtools >=1.7
     - bcftools >=1.7
+    - openssl =1.0
```

Comment:
You can remove the `=1.0` part, pinning is done automatically for openssl via the conda-forge-pinning package.